src/util/duration.cpp

```diff
@@ -17,6 +17,9 @@ static const qint64 kSecondsPerDay = 24 * kSecondsPerHour;
 
 } // namespace
 
+// static
+QString DurationBase::kCentisecondSeperator = QString::fromUtf8("\u2009");
```

Comment:
This requires a comment. 
// Unicode for THIN SPACE

I am also a bit confused why it works. The UTF8 character is 0xe28089 
"\u2009" should be already UTF-16, the internal codec of QString. Does the compiler turns it into utf8 and than fromUtf8 back at rune time? 
Did you try QChar(0x2009); ?